identifiers/country-ie/README.md

```diff
@@ -0,0 +1,6 @@
+# OCD Division Identifier
+
+This folder list the following administrative divisions of Ireland:
+
+* Lower House Constituencies [ref](https://en.wikipedia.org/wiki/D%C3%A1il_constituencies). Used type: **ed**.
```

Comment:
```suggestion
* Lower House Constituencies [ref](https://en.wikipedia.org/wiki/D%C3%A1il_constituencies).
```

You can remove this (here and below) once the `Type` section is added above.

Reply:
Followed your suggestion. Send an update. Please have a look.